packages/MarkdownEditor-Core.package/MarkdownHeading.class/class/attributesFor..st

```diff
@@ -0,0 +1,11 @@
+text attributes
+attributesFor: aString
+	
+	| headingLevel attributes |
+	headingLevel := self levelOf: aString.
+	attributes := OrderedCollection new.
+	((headingLevel + 1) isDivisibleBy: 2) ifTrue: [attributes add: TextEmphasis underlined].
+	(headingLevel between: 1 and: 2) ifTrue: [attributes add: (TextFontReference toFont: (TextStyle default copy fontOfSize: 26))].
```

Comment:
The following lines (or at least the font sizes) could be extracted into a method containing a constant. :blush:

Reply:
Yep, noted :D